docs-ref-conceptual/eclipse/java-sdk-azure-get-started-eclipse.md

```diff
@@ -0,0 +1,415 @@
+
+[Create a service principal using the Azure CLI 2.0](/cli/azure/create-an-azure-service-principal-azure-cli) and capture the output. Provide a [secure password](https://docs.microsoft.com/azure/active-directory/active-directory-passwords-policy) in the password argument instead of `MY_SECURE_PASSWORD`. Your password must be 8 to 16 characters and match at least 3 out of the 4 following criteria:
+
+* Include lowercase characters
+* Include uppercase characters
+* Include numbers
+* Include one of the following symbols: @ # $ % ^ & * - _ ! + = [ ] { } | \ : ‘ , . ? / ` ~ “ ( ) ;
+
+
+```azurecli-interactive
+az ad sp create-for-rbac --name AzureJavaTest --password "MY_SECURE_PASSWORD"
+```
+
+Which gives you a reply in the following format:
+
+```json
+{
+  "appId": "a487e0c1-82af-47d9-9a0b-af184eb87646d",
+  "displayName": "AzureJavaTest",
+  "name": "http://AzureJavaTest",
+  "password": password,
+  "tenant": "tttttttt-tttt-tttt-tttt-tttttttttttt"
+}
+```
+
+Next, copy the following into a text file on your system:
+
+```text
+# sample management library properties file
+subscription=ssssssss-ssss-ssss-ssss-ssssssssssss
+client=cccccccc-cccc-cccc-cccc-cccccccccccc
+key=kkkkkkkkkkkkkkkk
+tenant=tttttttt-tttt-tttt-tttt-tttttttttttt
+managementURI=https\://management.core.windows.net/
+baseURL=https\://management.azure.com/
+authURL=https\://login.windows.net/
+graphURL=https\://graph.windows.net/
+```
+
+Replace the top four values with the following:
+
+- subscription: use the *id* value from `az account show` in the Azure CLI 2.0.
+- client: use the *appId* value from the output taken from a service principal output.
+- key: use the *password* value from the service principal output.
+- tenant: use the *tenant* value from the service principal output.
+
+Save this file in a secure location on your system where your code can read it. You may use this file for future code so it's recommended to store it somewhere external to the application in this article.
+
+Set an environment variable `AZURE_AUTH_LOCATION` with the full path to the authentication file in your shell.
+
+```bash
+export AZURE_AUTH_LOCATION=/Users/raisa/azureauth.properties
+```
+
+If you're working in a windows environment, then you want to add the variable to your system properties. Open powershell and, after replacing the second variable with the path to your file, enter the following command:
+
+```powershell
+[Environment]::SetEnvironmentVariable("AZURE_AUTH_LOCATION", "C:\<fullpath>\azureauth.properties", "Machine")
+```
+
+## Create a new Maven project
+
+> [!NOTE]
+> This guide uses Maven build tool to build and run the sample code, but other build tools such as Gradle also work with the Azure libraries for Java. 
+
+Open Eclipse, select File > New. In the new window that appears open the Maven folder then select Maven Project. 
```

Comment:
Need some commas:

In the new window that appears, open the Maven folder, then select Maven Project.

Reply:
Make any menu commands, buttons, etc., caps and bold, e.g.:

**File -> New**

select **Next**